_includes/releases/v23.1/v23.1.0.md

```diff
@@ -0,0 +1,518 @@
+   </td>
+   <td><p><a href="../v23.1/fips-compliance.html">FIPS-ready binaries</a> and Docker images are available for CockroachDB 23.1.0 and above. Federal Information Processing Standards (FIPS) 140-2 is a standard used to approve cryptographic modules by the U.S. and Canadian governments for systems maintained by relevant agencies and organizations working with them for the purposes of encrypting and decrypting sensitive data.
+<p>FIPS-ready CockroachDB binaries are designed for workloads that require FIPS 140-2. FIPS-ready CockroachDB delegates cryptographic operations to the OpenSSL library available on the host operating system, rather than Go's cryptographic libraries. We recommend that OpenSSL has a FIPS 140-2 certificate. FIPS mode must be enabled in the Linux kernel to ensure that FIPS 140-2 is enforced by the operating system.
+   </td>
+  </tr>
+  <tr>
+   <td><p>Support Oauth authentication protocol for changefeeds </p>
+   </td>
+   <td><p>See this item in Change Data Capture (Changefeeds). 
+   </td>
+  </tr>
+  <tr>
+   <td><p>Support encrypted backups with keys stored in Azure Key Vault </p>
+   </td>
+   <td><p>You can now <a href="https://www.cockroachlabs.com/docs/stable/take-and-restore-encrypted-backups.html">take and restore encrypted backups</a> using RSA keys stored in Azure Key Vault.
+   </td>
+  </tr>
+  <tr>
+   <td><p>Expand on External Connections for Changefeeds </p>
+   </td>
+   <td><p>See the Changefeeds section for more information. 
+   </td>
+  </tr>
+  <tr>
+   <td><p>New fine-grained system privilege to view all jobs</p>
+   </td>
+   <td><p>The new <code><a href="../v23.1/security-reference/authorization.html#supported-privileges">VIEWJOB system privilege</a></code> allows a user to view all jobs when running commands like <code>SHOW JOBS</code>, without granting additional capabilities. This helps ensure alignment with the principle of least privilege by removing the need to assign broader privileges or role options like <code>CONTROLJOB</code>.
+   </td>
+  </tr>
+  </tbody>
+</table>
+
+<h4 id="v23-1-0-recovery-and-io">Recovery and I/O</h4>
+
+<h5>Change data capture (Changefeeds)</h5>
+
+<table>
+  <thead>
+   <tr>
+   <th>Feature</th>
+   <th>Description</th>
+   </tr>
+  </thead>
+  <tbody>
+  <tr>
+   <td><p>Create scheduled exports using changefeeds </p>
+   </td>
+   <td><p>Changefeeds can offer benefits over existing export functionality for customers who need higher levels of scale and observability. You can now create <a href="../v23.1/export-data-with-changefeeds.html">Changefeeds as Export</a> functionality.
+   </td>
+  </tr>
+  <tr>
+   <td><p>Use a webhook as a changefeed sink</p>
+   </td>
+   <td><p>The use of a <a href="../v23.1/changefeed-sinks.html#webhook-sink">webhook sink</a> to deliver changefeed messages to an arbitrary HTTPS endpoint has been promoted from Preview to GA.
+   </td>
+  </tr>
+  <tr>
+   <td><p>Add parquet format to changefeeds </p>
```

Comment:
Parquet format for changefeeds needs to be removed, we don't want this to be public yet.

Reply:
There's a known limitation I added for Parquet (https://github.com/cockroachdb/docs/pull/16902/files) - should I remove?